pkg/subctl/cmd/export.go

```diff
@@ -114,3 +122,28 @@ func validateArguments(args []string) error {
 	}
 	return nil
 }
+
+func weightAnnotationsFrom(rawWeights string) (map[string]string, error) {
+	weightClusterTuples := strings.Split(rawWeights, ",")
+	annotations := make(map[string]string, len(weightClusterTuples))
+	var err error
+
+	for _, weightClusterString := range weightClusterTuples {
+		weightClusterTuple := strings.Split(weightClusterString, ":")
+		if len(weightClusterTuple) != 2 {
+			err = fmt.Errorf("cluster weight tuple melformed %q", weightClusterString)
+			break
+		}
+		clusterID := weightClusterTuple[0]
+		weightStr := weightClusterTuple[1]
+		// Validating integer support only
+		if _, err := strconv.ParseInt(weightStr, 0, 64); err != nil {
+			err = fmt.Errorf("Error: %v parsing the weight %q for cluster %q, it is probably not of intenger format for the tuple %q ", err, clusterID, weightStr, weightClusterString)
+			break
+		}
+
+		annotations["lighthouse-load-balancer.submariner.io/weight"+"/"+clusterID] = weightStr
```

Comment:
To be consistent with other LH annotations:

```suggestion
		annotations["lighthouse.submariner.io/load-balancer-weights"+"/"+clusterID] = weightStr
```